src/components/arbeidsforhold/Arbeidsforhold.tsx

```diff
@@ -67,9 +70,9 @@ function ArbeidsforholdComponent(
 
   return (
     <div ref={ref} tabIndex={-1} aria-invalid={unansweredFaktumId === faktum.id}>
+      {/* {faktumTexts?.description && <PortableText value={faktumTexts.description} />} */}
```

Comment:
Note to self, den skal vi bare fjerne? Og bruker tekster fra sanity (dynamisk tekst).